tests/issues/issue-1923.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'issue-new-A' => [
```

Comment:
please rename the test to `issue-1923-A`.

Reply:
Fixed